VS filter: testpath:TestAssembly:SwamiQAWorks/TestClass:SwamiQAWorks.Features.ContactUsFeature/TestMethod:ValidSubmission
Discovering target: Default
Discovering tests from assembly: C:\Users\swaminathan\Documents\Visual Studio 2015\Projects\SwamiQAWorks\SwamiQAWorks\bin\Debug\SwamiQAWorks.dll
Appdomain Disposed
Test Suite Dump
#0: SwamiQAWorks/Default - 
#1: Default - 
#3: SwamiQAWorks - 
#4: SwamiQAWorks.Features.ContactUsFeature - 
#5: ValidSubmission - TestAssembly:SwamiQAWorks/TestClass:SwamiQAWorks.Features.ContactUsFeature/TestMethod:ValidSubmission
End: Test Suite Dump -- X
Scheduler: considering #5 for thread #0
Scheduler: using #5 for thread #0
Test #5 is scheduled for thread #0
Thread #0: creating new test thread executor for SwamiQAWorks, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null (target: Default)
Loading SpecRun type 'TechTalk.SpecRun.Framework.TestAssemblyExecutor' from: C:\Users\swaminathan\AppData\Local\Temp\VisualStudioTestExplorerExtensions\SpecRun.Runner.1.3.0\tools\TechTalk.SpecRun.Framework.Executor.dll
Test #5/0 is started on thread #0
T:Test environment set to staging
T:Given I am on the Contact Us Page
T:-> done: ContactUsSteps.GivenIAmOnTheContactUsPage() (2.3s)
T:When I send a message "please contact me I want to find out more " as "j.Bloggs6"and with email address "j.Blogge@qaworks.com"
T:-> error: Exception has been thrown by the target of an invocation.
T:Then the message should be sent successfully
T:-> skipped because of previous errors
Scheduler: dequeuing a rescheduled item: 5
Scheduler: considering #5 for thread #0
Scheduler: using #5 for thread #0
Test #5 is scheduled for thread #0
Test #5 is finished on thread #0 as Failed within 10.3348443 seconds
Test #5/1 is started on thread #0
T:Given I am on the Contact Us Page
T:-> done: ContactUsSteps.GivenIAmOnTheContactUsPage() (0.6s)
T:When I send a message "please contact me I want to find out more " as "j.Bloggs6"and with email address "j.Blogge@qaworks.com"
T:-> error: Exception has been thrown by the target of an invocation.
T:Then the message should be sent successfully
T:-> skipped because of previous errors
Scheduler: dequeuing a rescheduled item: 5
Scheduler: considering #5 for thread #0
Scheduler: using #5 for thread #0
Test #5 is scheduled for thread #0
Test #5 is finished on thread #0 as Failed within 1.2069852 seconds
Test #5/2 is started on thread #0
T:Given I am on the Contact Us Page
T:-> done: ContactUsSteps.GivenIAmOnTheContactUsPage() (0.8s)
T:When I send a message "please contact me I want to find out more " as "j.Bloggs6"and with email address "j.Blogge@qaworks.com"
T:-> error: Exception has been thrown by the target of an invocation.
T:Then the message should be sent successfully
T:-> skipped because of previous errors
Test #5 is finished on thread #0 as Failed within 1.407339 seconds
Appdomain Disposed
Total: 1 (test executions: 3)
Succeeded: 0
Ignored: 0
Pending: 0
Skipped: 0
Failed: 1
